lms/djangoapps/mobile_api/users/views.py

```diff
@@ -381,6 +382,17 @@ def list(self, request, *args, **kwargs):
 
         return response
 
+    # pylint: disable=attribute-defined-outside-init
+    @property
+    def paginator(self):
```

Comment:
How does this work with https://github.com/openedx/edx-platform/pull/33294/files#diff-e38b66be10e228f1728861821289555db4f2741b9057e1ecc70257d8ab52ddc8L318-L324 where the paginator class is set to `None`?  At the very least it seems like we might not be paginating the queryset itself which might make this less performant for users with a large number (1000+) of enrollments.

Reply:
Hi, I don't think there will be an issue, this is probably the only proven way to dynamically override pagination for an API based on the discussion: https://www.github.com/encode/django-rest-framework/issues/6397.

The alternative is to create a custom paginator, and put logic there, however because we are using just DefaultPagination or None, this might be an overhead in this case.

Reply:
Ah nice, can we add that link in the docstring or as a comment on this function for reference?